README.md

```diff
@@ -141,9 +141,9 @@ Some dependencies aren't available in the PyPI. They can be probably installed u
 - [flac](https://xiph.org/flac/)
 - [sox](http://sox.sourceforge.net/)
 
-PyPI installable dependencies are listed in the [requirements.txt](https://github.com/whipper-team/whipper/blob/master/requirements.txt) file and can be installed issuing the following command:
+PyPI installable dependencies are listed in the [setup.py](https://github.com/whipper-team/whipper/blob/master/setup.py) file and they are installed automatically when installing whipper with pip:
 
-`pip install -r requirements.txt`
+`python2 -m pip install -e .`
```

Comment:
Why the `-m` invocation instead of calling `pip` directly?

Reply:
sometime pip is not on the user's path, or it might be python3's pip.

This means that novice users cipypasting the instruction will have a lesser chance of encountering situations where things subtly break, at least in my opinion. (I can change if you want me to)